src/popover/src/Popover.js

```diff
@@ -87,7 +87,12 @@ export default class Popover extends Component {
     /**
      * When true, bring focus inside of the Popover on open.
      */
-    bringFocusInside: PropTypes.bool
+    bringFocusInside: PropTypes.bool,
+
+    /**
+     * Boolean indicating if clicking outside the dialog should close the dialog.
+     */
+    shouldNotCloseOnBodyClick: PropTypes.bool
```

Comment:
What about `shouldCloseOnExternalClick` to avoid the `shouldNot` negative?

Reply:
@mshwery You're welcome!

I was trying to make the default be falsy to prevent a breaking change.

If I go with `shouldCloseOnExternalClick`, I would need the default to be true, because I think the default behavior is that popover _does_ close on external click. Is my thinking correct there?

Reply:
That sounds right!

Reply:
Awesome. What are next steps for merging?

Reply:
@palmer-cl I can merge it in once we're all set with the naming!

Reply:
@mshwery re-read this comment: https://github.com/segmentio/evergreen/pull/558#discussion_r286265490

Im saying I **do not** think you want to name it what you proposed. It would be a breaking change to name it that and require default of true. Because by default, behavior should close on external click. We are adding a feature to NOT close on external click so the two options would be

1. `shouldCloseOnExternalClick` = true default (but this will be false on all current implementations)

2. `shouldNotCloseOnExternalClick` (like the user is proposing) = false default, but then passing it as true will give desired behavior.

Reply:
You can implement `shouldCloseOnExternalClick` in a non-breaking way.

`if (shouldCloseOnExternalClick === false)` + `static defaultProps = { shouldCloseOnExternalClick: true }` aka only when consumers explicitly pass it as false. That way when it's not explicitly passed at all we continue performing the current behavior, but if a consumer explicitly opts-out with `shouldCloseOnExternalClick={false}` we respect that.

Reply:
Roger that! I see what you're saying now. Updated in 24d05ac